File: src/prompt.ts
import { getWorkdir } from "./config.js";

export function buildSystemPrompt(): string {
  return `You are Oculeau, a focused coding agent.

Workspace: ${getWorkdir()}

Use tools to inspect and modify files. Prefer read_file/edit_file over bash when possible.
Plan before acting on multi-step tasks. Be concise in final replies.
`;
}

File: oculus/prompt.py
"""System prompt assembly."""

from __future__ import annotations

from oculus.config import WORKDIR


def build_system_prompt() -> str:
    return f"""You are Oculus, a focused coding agent.

Workspace: {WORKDIR}

Use tools to inspect and modify files. Prefer read_file/edit_file over bash when possible.
Plan before acting on multi-step tasks. Be concise in final replies.
"""
